src/qml/CoordinateLocator.qml

```diff
@@ -200,6 +235,40 @@ Item {
     }
   }
 
+  Shape {
+    id: snapToCommonAngleLines
+
+    property double endCoordX: 0
+    property double endCoordY: 0
+
+    visible: !!locator.commonAngleInDegrees || ( endCoordX !== 0 && endCoordY !== 0 )
+    width: parent.width
+    height: parent.height
+    anchors.centerIn: parent
+
+    // outer line
+    ShapePath {
+      strokeWidth: 4
+      strokeColor: "#fff"
```

Comment:
This is fully opaque? Might be worth reducing the opacity a bit. Same goes for inner line.

Reply:
0.5 opacity. Not what I expected, but at least does not "scream" like lines.
![image](https://github.com/opengisch/QField/assets/2820439/61ae4164-ee3c-4152-beaf-47ba7a31d911)

Reply:
@suricactus , I like it; do you?

Reply:
Done!